src/core_atmosphere/physics/mpas_atmphys_driver_radiation_sw.F

```diff
@@ -395,8 +395,14 @@ subroutine radiation_sw_from_MPAS(configs,mesh,state,time_lev,diag_physics,atm_i
  call mpas_pool_get_field(diag_physics,'m_hybi'    ,m_hybiField )
  call mpas_pool_get_field(diag_physics,'m_ps'      ,m_psField,time_lev)
 
- if (mpas_cpl%role_is(ROLE_INTEGRATE) .or. &
-     mpas_cpl%role_is(ROLE_RADIATION)) then
+ if (mpas_cpl%role_includes(ROLE_INTEGRATE)) then
+    !$acc update host(latCell, lonCell, aerosols, skintemp, snow, &
+    !$acc xice, xland, pin, ozmixm, sfc_albedo, sfc_emiss, cldfrac, &
+    !$acc o3clim, m_hybi, m_ps)
+ endif
+
+ if (mpas_cpl%role_includes(ROLE_INTEGRATE) .or. &
+     mpas_cpl%role_includes(ROLE_RADIATION)) then
```

Comment:
The original logic here was:
```fortran
 if (mpas_cpl%role_is(ROLE_INTEGRATE) .or. &
     mpas_cpl%role_is(ROLE_RADIATION)) then
```
which ensured that fields were transferred only if MPI tasks were performing an exclusive role (either INTEGRATE or RADIATION). The new logic,
```fortran
 if (mpas_cpl%role_includes(ROLE_INTEGRATE) .or. &
     mpas_cpl%role_includes(ROLE_RADIATION)) then
```
will transfer fields even in cases where MPI tasks are performing both INTEGRATE and RADIATION roles. Is this the intent?

Reply:
Browsing through the code, it looks like the pairings used to transfer fields will not even be generated in cases where MPI tasks are performing both INTEGRATE and RADIATION roles; see the [code from mpas_atm_core.F](https://github.com/MPAS-Dev/MPAS-Model/blob/atmosphere/develop-openacc/src/core_atmosphere/mpas_atm_core.F#L99-L105):
```fortran
      if (mpas_cpl % role_is(ROLE_INTEGRATE)) then
         cells_to_rad_handle = MPAS_coupler_create_pairing(mpas_cpl, MPAS_COUPLER_OUTGOING, ROLE_RADIATION, indexToCellID, nCellsSolve)
         cells_from_rad_handle = MPAS_coupler_create_pairing(mpas_cpl, MPAS_COUPLER_INCOMING, ROLE_RADIATION, indexToCellID, nCellsSolve)
      else if (mpas_cpl % role_is(ROLE_RADIATION)) then
         cells_to_rad_handle = MPAS_coupler_create_pairing(mpas_cpl, MPAS_COUPLER_INCOMING, ROLE_INTEGRATE, indexToCellID, nCellsSolve)
         cells_from_rad_handle = MPAS_coupler_create_pairing(mpas_cpl, MPAS_COUPLER_OUTGOING, ROLE_INTEGRATE, indexToCellID, nCellsSolve)
      end if
```